docs/api/client-request.md

```diff
@@ -49,6 +51,13 @@ following properties:
     [`request.followRedirect`](#requestfollowredirect) is invoked synchronously
     during the [`redirect`](#event-redirect) event.  Defaults to `follow`.
   * `origin` string (optional) - The origin URL of the request.
+  * `referrerPolicy` string (optional) - can be `""`, `no-referrer`,
```

Comment:
Empty string should not be a valid `referrerPolicy`, for using default value user should just omit this property (which equals to passing `undefined`).

Reply:
The empty string is required here per the [spec](https://w3c.github.io/webappsec-referrer-policy/#enumdef-referrerpolicy).